.circleci/config.yml

```diff
@@ -219,12 +219,6 @@ jobs:
     working_directory: ~/web3-onboard-monorepo/packages/mew
     steps:
       - node-build-steps
-  build-mew-wallet:
```

Comment:
Looks like these all got accidentally removed from the latest two releases

Reply:
@taylorjdawson yeah these should all be in here - I dont think anything should need to be removed from this file